operating_unit/models/operating_unit.py

```diff
@@ -17,6 +17,10 @@ class OperatingUnit(models.Model):
         'res.company', 'Company', required=True, default=lambda self:
         self.env['res.company']._company_default_get('account.account'))
     partner_id = fields.Many2one('res.partner', 'Partner', required=True)
+    user_ids = fields.Many2many(
+        'res.users', 'operating_unit_users_rel', 'poid', 'user_id',
```

Comment:
poid?

Reply:
Yes, that was the column of the original field. But I agree that is not correct and we should change both. I'll do the thing.

Reply:
Actaully, as the module is merged, I think it is better to do the thing in v13. I'll update the readme instead.